src/test_typing_extensions.py

```diff
@@ -5429,15 +5389,6 @@ def test_copy_and_pickle(self):
     def test_docstring(self):
         self.assertIsInstance(NamedTuple.__doc__, str)
 
-    @skipUnless(TYPING_3_8_0, "NamedTuple had a bad signature on <=3.7")
-    def test_signature_is_same_as_typing_NamedTuple(self):
-        self.assertEqual(inspect.signature(NamedTuple), inspect.signature(typing.NamedTuple))
```

Comment:
I deleted this test, as it no longer makes sense for us to have the same signature as `typing.NamedTuple` on <3.13, following #240